### DISCUSSION TITLE
After summoning the robot, the player will be stuck and read the card when logging in again after offline

### DISCUSSION THREAD

**Discussion by zjgzh001** [trickerer/Trinity-Bots#470] [Q&A]
Discovering a bug that prevents players from logging in to the game after summoning the robot and continuing to read the message 
I don't know if you've ever encountered it

**Comment by Russianranger** [comment]
I have never encountered this bug. Can you tell if it's a specific bot? Which ID? Do you have a screenshot of the message? Does it happen to everyone or just one person?

**Reply by zjgzh001** [reply]
After summoning NPCBOTS, go offline and then go online without taking a screenshot because it has been stuck in the loading interface. If NPCBOTS is not summoned, you can play or exit the game normally

**Comment by trickerer** [comment]
I might have, once. Not sure.
Occasionally something strange happens, client-side mostly. But that's extremely rare and the reason is never possible to find or deduct. Like once I had a problem where most models for bots wouldn't initialize and killed mobs didn't notify client right away, that was sudden and weird.

Thankfully, all of this is solved by restarting a server.

**Reply by zjgzh001** [reply]
After I entered the game, I was able to play the game normally without summoning robots, but every time I only needed to summon npcbots, I couldn't enter the game normally. Even after restarting the server, it would still get stuck in the loading interface, which is a headache for me. I plan to recompile and take a look
